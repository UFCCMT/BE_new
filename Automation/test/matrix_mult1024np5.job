#!/bin/bash
#SBATCH -D /ufrc/lam/johnson2319/NGEE/Automation
#SBATCH --job-name=matrix_mult1024np5.job
#SBATCH --mail-type=ALL
#SBATCH --mail-user=johnson@hcs.ufl.edu
#SBATCH --ntasks=5
#SBATCH --account=ccmt
#SBATCH --qos=ccmt
#SBATCH --mem-per-cpu=1gb
#SBATCH -t 00:5:00
#SBATCH -o test/matrix_mult1024np5.out
#SBATCH -e test/matrix_mult1024np5.err
 
module load intel/2016.0.109 openmpi/1.10.2
 
mpirun -np 5 ./mat_mult.o 1024
