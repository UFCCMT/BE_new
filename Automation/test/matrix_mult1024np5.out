./mat_mult.o
1024
Sending 256 rows to task 1
Sending 256 rows to task 2
Sending 256 rows to task 3
Sending 256 rows to task 4
Sent all the data!
Receiving data from task 1
Receiving data from task 2
Receiving data from task 3
Receiving data from task 4
( 5 )time spent 0.120000 sec
